UKXattrMetadataStore.h

```diff
@@ -1,37 +1,22 @@
 //
 //  UKXattrMetadataStore.h
 //  BubbleBrowser
+//	LICENSE: MIT License
```

Comment:
Nope. This license is not the MIT license. MIT requires credit, zlib doesn't. Can you please undo this change?

Reply:
Yeah. Sorry about that.